sdk/ai/azure-ai-projects/samples/agents/sample_agents_azure_ai_search.py

```diff
@@ -42,7 +42,8 @@
 print(conn_id)
 
 # Initialize agent AI search tool and add the search index connection id
-ai_search = AzureAISearchTool(index_connection_id=conn_id, index_name="myindexname")
+ai_search = AzureAISearchTool()
```

Comment:
The construction of AzureAISearchTool(index_connection_id=conn_id, index_name="myindexname") did not work but the add_index does?

@glharper please check if there is a bug.

Reply:
@jhakulin @e-straight Checked using built wheel from beta3 branch, not seeing a bug with the AzureAISearchTool constructor, it looks like the appropriate tool_resources object are added to the instantiated agent.